File: src/main/webapp/open-pricing-tracker/src/app/shared/models/settings.model.ts
import {autoserialize} from "cerialize";
/**
 * Provides the settings model
 */
export class SettingsModel {

  @autoserialize
  public id: number;

  @autoserialize
  public kubernetesName: string;

  @autoserialize
  public jenkinsName: string;

  @autoserialize
  public password: string;

  @autoserialize
  public jenkinsApiToken: string;
}

File: src/main/java/com/tracker/domain/settings/SettingsModel.java
package com.tracker.domain.settings;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * Provides the model that represented the settings
 */
@Getter
@Setter
public class SettingsModel {

    private int id;
    private String kubernetesName;
    private String jenkinsName;
    private String password;
    private String jenkinsApiToken;
}
